certified-connectors/DQ on Demand/apiDefinition.swagger.json

```diff
@@ -0,0 +1,7276 @@
+            "name": "query",
+            "x-example": "10 Downing Street sw1a",
+            "required": true,
+            "type": "string",
+            "description": "The Address to search for.",
+            "x-ms-visibility": "important",
+            "x-ms-summary": "Query"
+          },
+          {
+            "in": "query",
+            "name": "CountryIdentifier",
+            "x-example": "United Kingdom",
+            "required": true,
+            "type": "string",
+            "description": "A CountryIdentifer (GB, GBR, 44, United Kingdom, etc) for the country to search in.",
+            "x-ms-visibility": "important",
+            "x-ms-summary": "Country Identifier"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful Request.",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "status": {
+                  "description": "The HTTP Status code of the response.",
+                  "type": "integer",
+                  "x-ms-visibility": "advanced"
+                },
+                "message": {
+                  "description": "A message describing the request status.",
+                  "type": "string",
+                  "x-ms-visibility": "advanced"
+                },
+                "data": {
+                  "description": "The response.",
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "id": {
+                        "x-ms-visibility": "important",
+                        "type": "string",
+                        "x-nullable": true,
+                        "description": "The AddressId to use in the Retrieve call."
+                      },
+                      "label": {
+                        "x-ms-visibility": "important",
+                        "type": "string",
+                        "x-nullable": true,
+                        "description": "A preview of the address."
+                      },
+                      "subLabel": {
+                        "x-ms-visibility": "important",
+                        "type": "string",
+                        "x-nullable": true,
+                        "description": "A preview of the address."
+                      },
+                      "additionalInfo": {
+                        "type": "object",
+                        "x-ms-visibility": "advanced",
+                        "x-nullable": true,
+                        "additionalProperties": true
+                      }
+                    }
+                  }
+                }
+              }
+            },
+            "examples": {
+              "application/json": {
+                "status": 200,
+                "message": "success",
+                "data": [
+                  {
+                    "id": "VmtWb0K3XA-uaw4-f17gr32af45I",
```

Comment:
@lukeswanton Please remove the secret-look-a-like values from the swagger as per the Microsoft security guidelines.